test/tests/functional/pbs_mom_dynamic_resource_ext.py

```diff
@@ -145,3 +175,177 @@ def test_t6(self):
 
         # The job should run
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+
+    def test_t7(self):
+        """
+        Test for host level string resource
+        with string mom dynamic value
+        """
+
+        resc_name = "foo"
+        self.create_mom_resources(resc_name, "string", "h", "/bin/echo abc")
+
+        a = {'scheduling': 'True'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        # Submit a job that requests mom dynamic resource
+        attr = {"Resource_List." + resc_name: 'abc'}
+        j = Job(TEST_USER, attrs=attr)
+        jid = self.server.submit(j)
+
+        # The job should run
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+
+    def test_t8(self):
+        """
+        host level consumable float resource with mom dynamic value float
+        """
+        resc_name = "foo"
+        self.create_mom_resources(resc_name, "float", "nh", "/bin/echo 3")
+
+        a = {'scheduling': 'True'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        # Submit a job that requests mom dynamic resource
```

Comment:
update the comment saying "Submit a job that requests more value for the mom dynamic resource than available"